services/pdf-ms/lambdas/addPdfToCase.ts

```diff
@@ -35,7 +35,7 @@ export const main = async (event: Record<string, any>) => {
       .filter(c => c.SK !== SK)
       .sort((c1, c2) => c2.updatedAt - c1.updatedAt);
     const newestRelevantCase = sortedCases[0];
-    const comparison = compareCases(caseData, newestRelevantCase);
+    const comparison = getNewAndChangedValues(caseData, newestRelevantCase);
```

Comment:
noice word alert!
Data... RelevantCase....

Reply:
Just case cannot be used since it's a reserved keyword, that's why I'm using caseData. And yeah, maybe Relevant can be removed.